README.md

```diff
@@ -4,9 +4,10 @@ Recipe history for all things alcoholic.
 Here's a recipe template.
 
 # Beer 001
-## Author: [Someone]
+### Style
+### Author: [Someone]
 
-### Method: [2-vessel/3-vessel/BIAB]
+## Method: [2-vessel/3-vessel/BIAB]
 
 ### Target Brew Info
 | Stat | Value |
```

Comment:
Maybe it would make sense to add the water requirement up here? I'd like to have a place to document how much water I should be putting in my HLT.